frontend/src/lib/components/popups/Confirm.svelte

```diff
@@ -14,21 +14,25 @@
 </script>
 
 <div
-	class="absolute inset-0 z-20 flex justify-center items-baseline p-12 bg-grey bg-opacity-60"
+	class="fixed inset-0 z-20 flex justify-center items-baseline p-12 bg-grey bg-opacity-60"
 	transition:fade={{ duration: 200 }}
 	bind:clientHeight={paperHeight}
 >
-	<Paper class="pt-3 flex flex-col" elevation={7}>
+	<Paper class="p-6 flex flex-col" elevation={7}>
 		<Content>
-			<span>{message}</span>
+			<div class="mb-4">{message}</div>
 			<slot />
-			<div class="flex flex-row-reverse">
-				<Button style="margin-right: 10px" variant="outlined" on:click={() => dispatch('confirm')}>
-					{confirmText}
-				</Button>
+			<div class="flex m-auto justify-center">
 				<Button style="margin-right: 10px" variant="outlined" on:click={() => dispatch('cancel')}>
 					{cancelText}
 				</Button>
+				<Button
+					style="margin-right: 10px"
+					variant="unelevated"
```

Comment:
I would undo this because it's the only place we use this rn and the button is a different size which makes it look super weird.
Can fix this here if we want this style but would prefer a separate PR and using it as the confirm button in all dialogs.

Reply:
SG, we should change the color though it's hard to tell at a glance which is which.

Reply:
[ZEN-241 : update confirm button style](https://linear.app/zenoml/issue/ZEN-241/update-confirm-button-style)